backend/migrations/env.py

```diff
@@ -0,0 +1,98 @@
+import sys
+import asyncio
+from pathlib import Path as path
+from logging.config import fileConfig
+
+from alembic import context
+from sqlalchemy import pool
+from sqlalchemy.engine import Connection
+from sqlalchemy.ext.asyncio import async_engine_from_config
+
+BASE_DIR = path(path(__file__).parent, "../..").as_posix()
+sys.path.append(BASE_DIR)
+
```

Comment:
Why do we need this?

Reply:
Agree. It should be there

Reply:
this is here because of this project architecture, otherwise we get the error : ModuleNotFoundError: No module named 'backend'

Reply:
It should not be here. When you finish with all fixes, I will help you to escape of it